src/ray/common/scheduling/scheduling_test.cc

```diff
@@ -5,8 +5,143 @@
 #include <thread>
 
 #include "ray/common/scheduling/scheduling_ids.h"
+#include "ray/common/scheduling/scheduling_alg.h"
 using namespace std;
 
+/// Used to path empty vector argiuments.
```

Comment:
typos

Also, you can just initialize like `std::vector<int> vec;` and then use vec immediately as an empty vector